lunatrace/bsl/backend/src/config.ts

```diff
@@ -77,12 +94,45 @@ export function getEtlBucketConfig(): SbomHandlerConfig {
   };
 }
 
-export function getQueueHandlerConfig(): QueueHandlerConfig {
-  const queueName = checkEnvVar('QUEUE_NAME');
+export function getQueueHandlerConfig(): QueueHandlerWorkerConfig {
   const handlerName = checkEnvVar('QUEUE_HANDLER');
+
+  const DEFAULT_QUEUE_MAX_MESSAGES = 10;
+  const DEFAULT_QUEUE_VISIBILITY = 60;
+
+  const handlerConfigLookup: Record<QueueHandlerType, QueueHandlerConfig> = {
+    'process-webhook': {
+      maxMessages: 1,
+      visibility: DEFAULT_QUEUE_VISIBILITY * 5,
+      queueNameEnvVar: 'PROCESS_WEBHOOK_QUEUE'
+    },
+    'process-manifest': {
+      maxMessages: DEFAULT_QUEUE_MAX_MESSAGES,
+      visibility: DEFAULT_QUEUE_VISIBILITY,
+      queueNameEnvVar: 'PROCESS_MANIFEST_QUEUE'
+    },
+    'process-sbom': {
+      maxMessages: DEFAULT_QUEUE_MAX_MESSAGES,
+      visibility: DEFAULT_QUEUE_VISIBILITY,
+      queueNameEnvVar: 'PROCESS_SBOM_QUEUE'
+    },
+    'process-repository': {
+      maxMessages: DEFAULT_QUEUE_MAX_MESSAGES,
+      visibility: DEFAULT_QUEUE_VISIBILITY,
```

Comment:
You need to make this a longer time period than 60 seconds. You can see above on line 106 that I have this set to 5 minutes.

How long should it be set to? Well, how long do we expect a Git Clone operation to take?

Reply:
this is a good question. I am going to set this to 10 minutes for the moment. It might be too long, but I would rather have the window be on the longer side for the moment until we have metrics around it.